proto/tensor.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto2";
+
+message DimensionsProto {
+  repeated uint32 dimension = 1;
+}
+
+message TensorProto {
+  optional DimensionsProto dimensions = 1;
+  repeated float value = 2 [packed=true];
+}
+
+message ParameterProto {
+  optional DimensionsProto dimensions = 1;
+  repeated TensorProto value = 2;
+}
+
+message ModelProto {
+  repeated ParameterProto parameter = 1;
+  repeated ParameterProto lookup_parameter = 2;
+}
```

Comment:
Good catch. I've included the serialization of L2WeightDecay in the latest commit.